src/StoreKeeper/WooCommerce/B2C/Database/DatabaseConnection.php

```diff
@@ -121,4 +122,27 @@ public function prepare(QueryInterface $query)
             $query->getBindValues()
         );
     }
+
+    public static function formatToDatabaseDate(\DateTime $dateTime): string
+    {
+        $dateTime->setTimezone(new \DateTimeZone('UTC'));
```

Comment:
```suggestion
        $dateTime = clone $dateTime; 
        $dateTime->setTimezone(new \DateTimeZone('UTC'));
```

Otherwise you are changing the input paramter which is not a good practice, better to leave inputs uncahnged.

Reply:
Added cloning of the original datetime.